catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/ResultDAGConverter.java

```diff
@@ -151,16 +176,22 @@ public static void addFileNodeWithAttributes(DirectedAcyclicGraph<Node, Edge> gr
         Attribute pocAttr = metacard.getAttribute(Metacard.POINT_OF_CONTACT);
         if (pocAttr != null) {
             String pocString = String.valueOf(pocAttr.getValue());
-            if (pocString != null) {
+            if (StringUtils.isNotBlank(pocString)) {
                 addStringAttribute(graph, fileNode, NsiliConstants.CREATOR, pocString, orb);
+            } else {
+                addStringAttribute(graph, fileNode, NsiliConstants.CREATOR, "Alliance", orb);
```

Comment:
We may want to use SystemInfo.getSiteName() instead of `Alliance`, since it is configurable during the installation